src/index.js

```diff
@@ -1,9 +1,15 @@
 import { parse } from "babylon-lightscript";
 import { defaultImports, lightscriptImports, lodashImports } from "./stdlib";
 
+import { arrayStatementVisitor, objectStatementVisitor } from "./for-in";
+
 export default function (babel) {
   const { types: t } = babel;
 
+  // XXX: don't love this, but scrolling through a 2000 line file is maddening.
+  const forInArrayStatementVisitor = arrayStatementVisitor(babel);
```

Comment:
Bah, I feel your pain, but I'd really like to keep in one file for now... would that be ok?

FWIW, I'm planning to reorganize so that all node visitors are "top level" so you can code fold to level 2 and everything is accessible.